components/core/DiskMan.py

```diff
@@ -24,5 +24,8 @@ def remove_files(days, rate):
     if verbose:
         print("files to be removed", files)
     for file in files:
-        os.unlink(file[0])
+        try:
+            os.unlink(file[0])
+        except FileNotFoundError:
+            print('following file was not found on his path, removing match from database', file[0])
```

Comment:
Should be  
> on **this** path

Shall we go with capitalized log messages, we can change other places later on. 
> **F**ollowing file was not found on his path, removing the match from the database

Reply:
capitalization makes bit weird to read I think

Reply:
changes have done